ci-operator/step-registry/upi/install/vsphere/upi-install-vsphere-commands.sh

```diff
@@ -137,6 +137,53 @@ oc wait --for=condition=Updated mcp/worker --timeout=30m
 
 }
 
+function deploy_ccm() {
+set -x
```

Comment:
add the 2 leading whitespaces for the lines inside the function.

Reply:
done